string frame_id
ddrp/Objective[] objectives
ddrp/RegionObjectiveState[] region_objective_states

